source/process/release-process.md

```diff
@@ -123,9 +122,8 @@ Exceptions can be made by the release manager setting priority to "Highest" and
 7. Logistics:
     - Test RC fixes as they come in on CI servers
 8. Build: 
-    - Push next RC to acceptance after testing is complete and approved fixes merged, announces in Town Square on pre-release.mattermost.com/core
+    - Push next RC to acceptance after testing is complete and approved fixes merged, announces in Town Square. Build manager verifies with release manager before cutting any new RCs.
```

Comment:
Perhaps would read better worded as:
`Push next RC to acceptance after testing is complete and approved fixes are merged. Announce in Town Square. Build manager verifies with release manager before cutting any new RCs.`